lib/ApiResource.php

```diff
@@ -79,9 +79,19 @@ private static function _request($method, $path, $data = array()) {
     endif;
 
     if ($method === 'POST' || $method === 'PUT'):
-      curl_setopt($curl, CURLOPT_POST, 1);
+      if ($method === 'PUT'):
```

Comment:
i think this syntax has been around as an alternative. maybe not widely used! i always preferred it for some reason.